CoreImportExportPlugins/src/au/gov/asd/tac/constellation/plugins/importexport/delimited/DelimitedSourcePane.java

```diff
@@ -172,71 +174,60 @@ public DelimitedSourcePane(final DelimitedImportController importController) {
     }
 
     private void addFile(final DelimitedImportController importController) {
-        final FileChooser fileChooser = new FileChooser();
-        fileChooser.setInitialDirectory(defaultDirectory);
-
         final ImportFileParser parser = DelimitedSourcePane.this.importFileParserComboBox.getSelectionModel().getSelectedItem();
 
         if (parser != null) {
-            final ExtensionFilter extensionFilter = parser.getExtensionFilter();
-
-            if (extensionFilter != null) {
-                fileChooser.getExtensionFilters().add(extensionFilter);
-                fileChooser.setSelectedExtensionFilter(extensionFilter);
+            final FileFilter fileFilter = parser.getFileFilter();
+
+            if (fileFilter != null) {
+                FileChooser.openMultiDialog(getDelimitedImportFileChooser().setFileFilter(fileFilter)).thenAccept(optionalFiles -> optionalFiles.ifPresent(newFiles -> {
+                    Platform.runLater(() -> {
+                        DelimitedSourcePane.this.importFileParserComboBox.setDisable(true);
+
+                        final ObservableList<File> files = FXCollections.observableArrayList(fileListView.getItems());
+                        final StringBuilder sb = new StringBuilder();
+                        final String alertText = "The following files could not be parsed and have been excluded from the import set:\n";
+                        sb.append(alertText);
+
+                        for (final File file : newFiles) {
+                            // prevent adding the same file again
+                            if (fileAlreadyAdded(file)) {
+                                continue;
+                            }
+
+                            // Attempt to parse/preview, if a failure is detected don't add the file to the set of files to import.
+                            try {
+                                parser.preview(new InputSource(file), null, PREVIEW_LIMIT);
+                                files.add(file);
+                            } catch (final IOException ex) {
+                                // Append the name of each file that could not be imported.
+                                sb.append("\n");
+                                sb.append(file.getName());
+                                LOGGER.log(Level.INFO, "Unable to parse the file {0}, " + "excluding from import set.", new Object[]{file.toString()});
```

Comment:
This seems more like `WARNING` than `INFO`. Also you should be able to have merge those 2 strings together
i.e. `"...{0}, excluding..."` rather than `"...{0}, " + "excluding..."`